### ISSUE TITLE
Any way to control non-iCue devices that are detected by iCue?

### ISSUE THREAD

**Issue by krasatos** [tobinbrooke/icue2mqtt#8]
Hey there & thank you for this service, been looking for something similar for a while.

I have an ASUS aura motherboard that is currently being discovered and controlled by iCue.

Is there any way to send those devices over to HA?

![image](https://user-images.githubusercontent.com/1773624/114839546-91d6b500-9dde-11eb-9262-14579ef3e9cf.png)

**Comment by tobinbrooke** [comment]
Yeah, I have the same. I did look a while ago but at that point the iCue SDK did not allow it. I plan to try again at some point but I have time constraints for a few months

**Comment by krasatos** [comment]
> Yeah, I have the same. I did look a while ago but at that point the iCue SDK did not allow it. I plan to try again at some point but I have time constraints for a few months

Thank you, really looking forward to it.
If there's any way i can help out, please let me know (with logs or anything)
BTW, any ideas that might quickly fix this: #7 ?

**Comment by tobinbrooke** [comment]
There are a few variables. The iCue control is done using NuGet C# wrapper around the Corsair iCue C++ SDK. As a result, it depends if Corsair have released a new SDK and if the NuGet package has been updated to use it.
If both are updated then should be easy. If Corsair have done a release but not NuGet, I will have to find it and write my own wrapper. If neither are true then would have to see about adding control to Aura but given how buggy the Asus drivers are in a normal use case I don't foresee that ending well.

**Comment by tobinbrooke** [comment]
If you have experience with C# feel free to do a pull request or fork

**Comment by krasatos** [comment]
> If you have experience with C# feel free to do a pull request or fork

Zero knowledge unfortunately :(
